Translation corrections (#275)
07th-mod/higurashi-console-arcs: Update/yoig_014.txt

//rあの、…赤坂さん。そうおっしゃるならその、羽虫なら近くを飛んだだけでも殺せるほどの殺気は抑えていただけないでしょうか…？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "あの、…赤坂さん。そうおっしゃるならその、羽虫なら近くを飛んだだけでも殺せるほどの殺気は抑えていただけないでしょうか…？",

> 			NULL, "Umm... Akasaka-san. You may say that, but could you suppress that blood lust that could kill any insect that got too close to you...?", Line_Normal);

Reviewer: ```suggestion
			NULL, "Umm... Akasaka-san. You may say that, but could you suppress that bloodlust that could kill any insect that got too close to you...?", Line_Normal);
```

//rいや、…それよりもだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "いや、…それよりもだ。",

> 			NULL, "No... it was more importantly.", Line_Normal);

Reviewer: ```suggestion
			NULL, "No... more importantly.", Line_Normal);
```

//r詩音さんはそこから降りて、一見してそれとわかる風貌の男が、彼女を先導するように俺たちのもとへ歩み寄ってくる。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "詩音さんはそこから降りて、一見してそれとわかる風貌の男が、彼女を先導するように俺たちのもとへ歩み寄ってくる。",

> 			NULL, "Shion-san got out of there, and a man whose appearance betrayed his aura walking over to us as if guiding her. ", Line_Normal);

Reviewer: ```suggestion
			NULL, "Shion-san got out of there, and a man whose appearance betrayed his aura walked over to us as if guiding her. ", Line_Normal);
```

//rまったく、…昇天の直前に何を言い残したかと思ったら、そんなことかい。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "まったく、…昇天の直前に何を言い残したかと思ったら、そんなことかい。",
			NULL, "Sheesh... that's seriously what you chose to say at the bitter end?", Line_Normal);
	ClearMessage();

//rあの時は私が助けてやったのに、これまでずっとお節介焼いて…子ども扱いしてくれちゃって。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "あの時は私が助けてやったのに、これまでずっとお節介焼いて…子ども扱いしてくれちゃって。",
			NULL, "Even though she helped out back then, she's always butting in where she doesn't belong... treating me like a child.", Line_Normal);
	ClearMessage();

//r…私はもうあんたよりもずっと年上で、大人になってるのにさ？　最後の言葉が、そんな青臭いセリフだなんて、…まったく……。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…私はもうあんたよりもずっと年上で、大人になってるのにさ？　最後の言葉が、そんな青臭いセリフだなんて、…まったく……。",

> 			NULL, "I'm an adult now, you know? Much older than you were. Seriously... I can't believe something so naive were your last words...", Line_Normal);

Reviewer: ```suggestion
			NULL, "I'm an adult now, you know? Much older than you were. Seriously... I can't believe your last words were something so naive...", Line_Normal);
```

//r次なんて、…悲しい言葉だ。いなくなったことを思い知らされるから…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "次なんて、…悲しい言葉だ。いなくなったことを思い知らされるから…。",

> 			NULL, "And the word "next"... made me sad. Because you reminded me that you're already gone.", Line_Normal);

Reviewer: ```suggestion
			NULL, "And the word \"next\"... made me sad. Because you reminded me that you're already gone.", Line_Normal);
```
